docs/app/pages/components/components-sections/wizard/marquee-wizard/marquee-wizard.component.html

```diff
@@ -262,6 +262,10 @@ <h1 class="m-t-nil">Sixth step title</h1>
         If set to
         <code>true</code> the 'Next' button will appear disabled and will not respond to clicks.
     </tr>
+    <tr uxd-api-property name="stepInvalid" type="boolean">
```

Comment:
It looks like this may be the wrong property you are documenting, your test is using `nextInvalid` as the input.

It might be good to update the description to be something like:

"If set to `true` the 'Next' button will become disabled when the current step is invalid."